PreRun: Creating required directories
Directory Outputs/ already exists
Directory log/ already exists
PreRun: generating experiment config file
Using cpu device
configuring data loaders
configuring model - UNET 3+
saving model summary
Traceback (most recent call last):
  File "/home/blue/projects/MoNuSeg/main.py", line 537, in <module>
    main()
  File "/home/blue/projects/MoNuSeg/main.py", line 285, in main
    saveTorchSummary(model, input_size=(3, 256, 256), path=config["expt_dir"]+"modelSummary.txt")
  File "/home/blue/projects/MoNuSeg/auxilary/utils.py", line 104, in saveTorchSummary
    summary(model, input_size, )
  File "/home/blue/projects/MoNuSeg/monuSegEnv/lib/python3.10/site-packages/torchsummary/torchsummary.py", line 72, in summary
    model(*x)
  File "/home/blue/projects/MoNuSeg/monuSegEnv/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/blue/projects/MoNuSeg/networkModules/modelUnet3p.py", line 217, in forward
    h2 = self.maxpool1(h1)
  File "/home/blue/projects/MoNuSeg/monuSegEnv/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1538, in _call_impl
    result = forward_call(*args, **kwargs)
  File "/home/blue/projects/MoNuSeg/networkModules/conv_modules_unet3p.py", line 77, in forward
    return F.conv2d(x, self.weight, stride=1, groups=x.size(1))  # Apply blur
RuntimeError: Given groups=16, expected weight to be at least 16 at dimension 0, but got weight of size [3, 1, 3, 3] instead